src/glyphs.js

```diff
@@ -273,4 +351,26 @@ router.post('/:_id/messages/:num', (req, res) => {
   }
 });
 
+// This route is used to import messages from a spreadsheet (CSV)
+router.use(formidable()).post('/import/', (req, res) => {
+  const processed = processImportRequest(req);
+  let response = [];
+
+  if (processed.errors.length > 0) {
+    res.json({
+      'errors': processed.errors
+    });
+  } else {
+    if (req.fields.type === "overwrite") overwriteMessages(req.files[""].path);
+
+    let first = true;
+    lineReader.eachLine(req.files[""].path, function(line) {
```

Comment:
It feels really weird to me that `overwriteMessages` does the line reading but `importMessages` has it happen inside this function.  Shouldn't they both deviate the same amount?